library/src/main/java/dev/hichamboushaba/suspendactivityresult/Helpers.kt

```diff
@@ -90,4 +92,18 @@ suspend fun ActivityResultManager.openDocumentTree(startingLocation: Uri? = null
 @RequiresApi(19)
 suspend fun ActivityResultManager.createDocument(fileName: String): Uri? {
     return requestResult(CreateDocument(), fileName)
-}
\ No newline at end of file
+}
+
+/**
+ * see [StartIntentSenderForResult]
+ */
+suspend fun ActivityResultManager.takeActivityResult(intentSenderRequest: IntentSenderRequest): ActivityResult? {
```

Comment:
Let's rename this to the same name that the default function uses: [`startIntentSender`](https://developer.android.com/reference/android/app/Activity.html#startIntentSender-android.content.IntentSender-android.content.Intent?-kotlin.Int-kotlin.Int-kotlin.Int-)

Reply:
fix